### DISCUSSION TITLE
How to use custom pickers and defaults?

### DISCUSSION THREAD

**Discussion by RAprogramm** [uga-rosa/ccc.nvim#108] [Q&A] [answered]
```lua
{
    pickers = {
      ccc.picker.custom_entries(require('neviraide-ui.themes.colors').palette),
    },
    win_opts = {
      border = vim.g.b,
    },
    highlighter = {
      auto_enable = true,
      excludes = { 'neo-tree' },
      lsp = true,
    },
  }
```

if i do like this - it works for custom colors but not working for '#ff0000' format

![240312_14h36m48s_screenshot](https://github.com/uga-rosa/ccc.nvim/assets/70325462/453b2571-947f-4b70-bffe-e8c07e5a8ab8)

![240312_14h37m54s_screenshot](https://github.com/uga-rosa/ccc.nvim/assets/70325462/332c5c40-c640-4e42-a807-0eb8a9af51a1)

how to make highlight after `color.<some_color>` and `'#ff0000'` both?

**Comment by uga-rosa** [comment, marked as answer]
You should read the help. Default values are on `:h ccc-option-pickers`.

```lua
{
  pickers = {
    ccc.picker.hex,
    ccc.picker.css_rgb,
    ccc.picker.css_hsl,
    ccc.picker.css_hwb,
    ccc.picker.css_lab,
    ccc.picker.css_lch,
    ccc.picker.css_oklab,
    ccc.picker.css_oklch,
    ccc.picker.custom_entries(require('neviraide-ui.themes.colors').palette),
  },
}
```

**Comment by RAprogramm** [comment]
thank you for answer.
one more thing...does it possible to show custom entries only after variable which define colors?
for example if if define `white = '#ffffff'` not highlight it, but if i use this , for example, `{fg = color.white}` - highlight it

**Reply by uga-rosa** [reply]
Not possible.
This plugin does not parse syntax. It cannot do more than simple string parsing.
